ml/src/main/scala/org/apache/spark/ml/spark/models/svm/SVMModel.scala

```diff
@@ -28,6 +31,7 @@ object SVMModel {
     var interceptor: Double = .0
     var iterations: Int = 0
     var weights: Array[Double] = null
+    var numMeans: Array[Double] = null
```

Comment:
Using nulls in Scala is not a good practice at all. If a value is optional, use Option.
Using vars... well... it may be harder to eliminate